bin/compile

```diff
@@ -24,11 +24,7 @@ POPPLER_VERSION="21.08.0"
 POPPLER_BUILD_DIR="$BUILD_DIR/.heroku/poppler"
 
 mkdir $POPPLER_BUILD_DIR
-
-topic "Fetching Poppler $POPPLER_VERSION"
-
-curl -sL -O https://poppler.freedesktop.org/poppler-$POPPLER_VERSION.tar.xz | indent
-tar -xf poppler-$POPPLER_VERSION.tar.xz
+tar -xf $BUILDPACK_DIR/archive/poppler-$POPPLER_VERSION.tar.xz --directory $BUILDPACK_DIR
```

Comment:
Ok, I didn't notice the file above, and I was wondering what we are doing here 😂. For other reviewers, it's right above this file, here is a screenshot of it.  

<img width="700" alt="Screen Shot 2022-01-05 at 7 55 02 PM" src="https://user-images.githubusercontent.com/1326771/148310886-73d1759d-fe23-462f-8381-4ced032b0ba6.png">

I will approve this and assume Alex ran (or plans to do) a test run in our of non-prod apps.